src/routes/userRouter.ts

```diff
@@ -72,4 +72,15 @@ userRouter.post(
   userControllers.updateUserPicture,
 );
 
+// Get posts by user
+
+userRouter.get(
+  '/:userId/posts',
+  validator([
+    param('userId').isUUID(4),
+  ]),
+  userControllers.getUserPosts,
+
```

Comment:
This is a bit nit-picking (sorry), but let's remove the two unnecessary empty lines to keep it consistent with the rest of the file. This is a very small thing, but eventually when you work a lot on a project and not keep a consistent code style it can make it a bit harder to read.

Reply:
great! thanks for reminding me.